src/components/v5/common/ActionSidebar/hooks/useGetActionData.ts

```diff
@@ -124,20 +124,33 @@ export const useGetActionData = (transactionId: string | undefined) => {
         };
       case ColonyActionType.ColonyEdit:
       case ColonyActionType.ColonyEditMotion: {
-        const modifiedTokens =
-          action.pendingColonyMetadata?.modifiedTokenAddresses?.added?.map(
-            (addedToken) => ({
-              token: addedToken,
-            }),
-          );
+        const changelog = action.colony.metadata?.changelog?.find(
+          ({ transactionHash }) => transactionHash === action.transactionHash,
+        );
+        const { haveTokensChanged } = changelog || {};
+        const modifiedTokens = isMotion
+          ? action.pendingColonyMetadata?.modifiedTokenAddresses
+          : action.colony.metadata?.modifiedTokenAddresses;
+        const newTokens = modifiedTokens?.added?.map((addedToken) => ({
+          token: addedToken,
+          isNew: true,
+        }));
+        const removedTokens = modifiedTokens?.removed?.map((addedToken) => ({
```

Comment:
`modifiedTokens?.removed` has incorrect data, as it populates the list with tokens that have not been removed

![Screenshot from 2023-12-19 16-04-39](https://github.com/JoinColony/colonyCDapp/assets/1193222/b34591cd-d7d0-4d87-bd7c-39def2de3bfd)

Also, the list shows a token two times

Reply:
@rdig - @arrenv said that I should use this field to get the information about removed tokens. If this is incorrect, then can you tell me what should be used instead?